main.tf

```diff
@@ -10,6 +10,10 @@ provider "aws" {
 }
 
 data "aws_region" "current" {}
+data "aws_route53_zone" "current" {
```

Comment:
If the block requires config (like in this case) I prefer to separate to from other resources with a blank line.